.env

```
@@ -0,0 +1,3 @@
+DB_URL=jdbc:mysql://localhost:3306/repit
+DB_USERNAME=root
+DB_PASSWORD=1234abcd
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)